managed/src/main/java/com/yugabyte/yw/controllers/CustomerConfigController.java

```diff
@@ -57,4 +63,31 @@ public Result delete(UUID customerUUID, UUID configUUID) {
   public Result list(UUID customerUUID) {
     return ApiResponse.success(CustomerConfig.getAll(customerUUID));
   }
+
+  public Result edit(UUID customerUUID, UUID configUUID) {
+    JsonNode formData =  request().body().asJson();
+    ObjectNode errorJson = configValidator.validateFormData(formData);
+    if (errorJson.size() > 0) {
+      return ApiResponse.error(BAD_REQUEST, errorJson);
+    }
+
+    errorJson = configValidator.validateDataContent(formData);
+    if (errorJson.size() > 0) {
+      return ApiResponse.error(BAD_REQUEST, errorJson);
+    }
+    CustomerConfig customerConfig = CustomerConfig.get(customerUUID, configUUID);
+    if (customerConfig == null) {
+      return ApiResponse.error(BAD_REQUEST, "Invalid configUUID: " + configUUID);
+    }
+    CustomerConfig config = CustomerConfig.get(configUUID);
+    JsonNode data = Json.toJson(formData.get("data"));
+    if (data != null && data.get("BACKUP_LOCATION") != null) {
```

Comment:
we are using the string instead of constants lot of places. Including other tests and utils file for masking. I will create another issue to fix these separatly.